index.js

```diff
@@ -22,16 +22,25 @@ const getDefaultBaseReleaseTag = async () => {
 (async function() {
   try {
     const token = core.getInput('token');
-    octokit = github.getOctokit(token);
+    const runOnPR = core.getInput('run-on-pr') === 'true';
+    const continueOnError = core.getInput('continue-on-error') === 'true';
+
+    const octokit = github.getOctokit(token);
     console.log("Initiated octokit");
 
-    const headReleaseTag = await getHeadReleaseTag()
-    console.log("Head release tag: ", headReleaseTag);
+    let baseReleaseTag, headReleaseTag;
+    if (runOnPR && context.payload.pull_request) {
```

Comment:
You can check the github context here to find out if the run is started from a release or a PR. IMO a little bit cleaner than the separate argument.